Source/Core/InputCommon/ControllerEmu/ControllerEmu.h

```diff
@@ -86,6 +88,8 @@ class EmulatedController
   void UpdateReferences(ciface::ExpressionParser::ControlEnvironment& env);
 
 private:
+  void GenerateTextures(IniFile::Section* sec);
+  InputCommon::DynamicInputTextureManager* m_dynamic_input_tex_config_manager = nullptr;
```

Comment:
I've been thinking about this for a couple days and don't have any ideas for a better design.  I'm all ears if someone has an idea.  It should be safe though, since `InputConfig` owns both the `ControllerEmu` and the `DynamicInputTextureManager`.